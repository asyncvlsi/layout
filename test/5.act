export defcell mycell (bool? in[5]; bool! out)
{
   prs * {
   in[0]<15> & (in[1]<11> & in[2]<23> | in[3]<11> & in[4]<23>) -> out-
   ~in[0]<21> & ~in[2]<21> & ~in[4]<21> -> out+
   }
}

defproc test()
{
  mycell x;
}

test t;
