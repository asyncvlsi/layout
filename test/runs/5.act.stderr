WARNING: lefdef.pin_layer (2) is a horizontal metal layer.
	[default pin locations are in a line at the top/bottom of the cell]
